editor/static/js/editor.js

```diff
@@ -415,9 +415,8 @@ $(document).ready(function() {
         this.setTab = function(id) {
             return function() {
                 var tab = ei.getTab(id);
-
                 ei.currentTab(tab);
-                ei.set_ignored_publishing_criteria(false);
+                ei.ignored_publishing_criteria(false);

```

Comment:
This doesn't work - most of the time, the tab is changed by modifying the `currentTab` observable directly. 
Anyway, this is not the right place to add a side-effect which isn't part of the tab system - instead, there should be a computed observable whose only job is to watch the current tab and set `ignored_publishing_criteria` if the tab changes.